adapter/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
 	implementation 'com.github.fwcd.kotlin-language-server:shared:229c762a4d75304d21eba6d8e1231ed949247629'
 	// The Java Debug Interface classes (com.sun.jdi.*)
 	implementation files("${System.properties['java.home']}/../lib/tools.jar")
+	// For CommandLineUtils.translateCommandLine
+	implementation 'org.codehaus.plexus:plexus-utils:3.3.0'
```

Comment:
If possible, I would like to avoid introducing a new dependency here. See the note below for more details.